src/main/java/uk/gov/hmcts/ccd/domain/service/common/CaseStateUpdateService.java

```diff
@@ -3,34 +3,43 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import java.util.List;
 import java.util.Map;
-import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
 import javax.inject.Inject;
 import org.springframework.stereotype.Service;
-import uk.gov.hmcts.ccd.domain.casestate.EnablingConditionSorter;
+import uk.gov.hmcts.ccd.domain.enablingcondition.PrioritiseEnablingCondition;
 import uk.gov.hmcts.ccd.domain.model.definition.CaseDetails;
 import uk.gov.hmcts.ccd.domain.model.definition.CaseEventDefinition;
 import uk.gov.hmcts.ccd.domain.model.definition.EventPostStateDefinition;
 
 @Service
 public class CaseStateUpdateService {
```

Comment:
```suggestion
public class CasePostStateService {
```
this class is not really doing any case update itself right?